assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
+                "14": ["_ILBF2DME", "_INTGAM327", ["_GAM327036", "A30+|A70-"], ["_GAM327032", "A35+|A70-"], ["_GAM327029", "A70"], "_GAM327019", "_GAM327015", "_GAM327014", "_GAM327POL118"],
+                "32": [["_ILF0.5DME", "A12+"], "_ILF2.1DME", "_ILF3.5DME", ["_INTGAM319", "A30+|A70-"], ["_GAM319038", "A35+|A70-"], ["_GAM319035", "A60+|A70-"], ["_GAM319031", "A70"], "_GAM319019", "_GAM319018", "_GAMR319INTPOLR118", "_GAM15DME", "DOPEK"]
             },
             "body": [],
             "exitPoints": {
-                "POL": [["POL", "A70"]]
+                "DOPEK": ["DOPEK"]
             },
             "draw": [
-                ["D082N", "D082J", "POL"]
+                ["_ILBF2DME", "_INTGAM327", "_GAM327036", "_GAM327032", "_GAM327029", "_GAM327019", "_GAM327015", "_GAM327014", "_GAM327POL118", "DOPEK"],
+                ["_ILF0.5DME", "_ILF2.1DME", "_ILF3.5DME", "_INTGAM319", "_GAM319038", "_GAM319035", "_GAM319031", "_GAM319019", "_GAM319018", "_GAMR319INTPOLR118", "_GAM15DME", "DOPEK"]
             ]
         },
-        "NELSA": {
-            "icao": "NELSA",
-            "name": "Nelsa Three Whiskey",
-            "suffix": {"32":"3W"},
+        "LAMIX": {
+            "icao": "LAMIX",
+            "name": "Lamix",
+            "suffix": {"14":"2X", "32":"2W"},
             "rwy": {
-                "32": ["05DME", "21DME", "35DME", ["D052L", "A30"], ["D052K", "A35"], ["D272J", "A35"]]
+                "14": ["_ILBF2DME", "_INTGAM327", ["_GAM327036", "A30+|A70-"], ["_GAM327032", "A35+|A70-"], ["_GAM327029", "A70"], "_GAM327019", "_GAM327015", "_GAM327014", "_GAM327POL118"],
+                "32": [["_ILF0.5DME", "A12+"], "_ILF2.1DME", "_ILF3.5DME", ["_INTGAM319", "A30+|A70-"], ["_GAM319038", "A35+|A70-"], ["_GAM319035", "A60+|A70-"], ["_GAM319031", "A70"], "_GAM319019", "_GAM319018", "_GAMR319INTPOLR118", "_GAM15DME"]
             },
-            "body": [],
+            "body": ["DOPEK"],
             "exitPoints": {
-                "NELSA": [["NELSA", "A70"]]
+                "LAMIX": ["LAMIX"]
             },
             "draw": [
-                ["05DME", "21DME", "35DME", "D052L", "D052K", "D272J", "NELSA"]
+                ["DOPEK", "LAMIX"]
             ]
         },
-        "DOPEK": {
-            "icao": "DOPEK",
-            "name": "Dopek",
-            "suffix": {"32":"2W", "14":"2X"},
+        "NELSA3W": {
+            "icao": "NELSA3W",
+            "name": "Nelsa Three Whiskey",
             "rwy": {
-                "32": ["05DME", "21DME", "35DME", ["R255", "A30"], ["GAM38", "A35"], ["GAM35", "A60"], ["GAM31", "A70"], "GAM19", "GAM18", "GAM15"],
-                "14": ["D140B", ["GAM36", "A30"], ["GAM32", "A35"], ["GAM29", "A70"], "GA19A", "GA15A", "GAM14"]
+                "32": [["_ILF0.5DME", "A12+"]]
             },
-            "body": [],
+            "body": ["_ILF2.1DME", "_ILF3.5DME", ["_POL053012", "A30+|A70-"], "_POL053011" ,["_POL053010", "A35+|A70-"], "_INTLBA272"],
```

Comment:
- `_ILF3.5DME` & `_POL053011` should probably be fly-over
- the `A70-`'s aren't needed because of the restriction at `NELSA`
- Fix comma between` _POL053011` & `_POL053010`